brianchandotcom/liferay-portal
# LPS-189476 Copy tag ratings and categories when coping

Follow up from https://github.com/liferay-lima/liferay-portal/pull/4444
IT's the same that https://github.com/liferay-lima/liferay-portal/pull/4452 but without the rebase
LPS-187728 is needed a rebase so > on hold

https://liferay.atlassian.net/browse/LPS-189476

To test it :
For all: Enable FF: LPS-182512

Go to global site
On this create a category and a tag
Create a Document with created category and tag
Go to document > dropdown > copy to
copy to Liferay site (home)
Go to Liferay site > Documents and Media
check document copied from global (edit) has the tag and the category
other test

Create an Asset Library
Connect to Liferay Site
On the asset library create a category and a tag
Create a Document with created category and tag
Go to document > dropdown > copy to
copy to Liferay site (home)
Go to Liferay site > Documents and Media
check document copied from asset library has the tag and the category
note : if copy between sites not related, only common categories and tags (from global or an common ancestor) is copied

@brianchandotcom resending this pull directly to you since after a few retries we confirmed this pull is not related to any of the failing tests in https://github.com/liferay-lima/liferay-portal/pull/4522. And these changes under the FF. Thanks.

/cc @AliciaGarciaGarcia
